src/api/costModelsProviders.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { PagedLinks, PagedMetaData } from './api';
+import {
+  ProviderAuthentication,
+  ProviderBillingSource,
+  ProviderCreatedBy,
+  ProviderCustomer,
+} from './providers';
+
+export interface ProviderCostModel {
+  name: string;
+  uuid: string;
+}
+
+export interface Provider {
+  uuid?: string;
+  name?: string;
+  type?: string;
+  authentication?: ProviderAuthentication;
+  billing_source?: ProviderBillingSource;
+  customer?: ProviderCustomer;
+  created_by?: ProviderCreatedBy;
+  created_timestamp?: Date;
+  cost_models?: ProviderCostModel[];
+}
+
+export interface Providers {
+  meta: PagedMetaData;
+  links?: PagedLinks;
+  data: Provider[];
+}
+
+export const enum ProviderType {
+  aws = 'aws',
+  azure = 'azure',
+  ocp = 'ocp',
+}
+
+export function fetchProviders(query: string) {
+  const queryString = query ? `?${query}` : '';
+  return axios.get<Providers>(`providers/${queryString}`);
```

Comment:
Should we be using the new `sources` API?

Reply:
I am not sure. I wasn't told that providers is deprecated and if so - `sources` doesn't have the `cost_models` attribute which I am relaying on in the cost models page.

//cc @chambridge can you bring more details about this?

Reply:
https://github.com/project-koku/koku/pull/1832 should resolve the issue. Just waiting for approval to merge.

Reply:
After talking with @chambridge - it was informed to me that PR https://github.com/project-koku/koku/pull/1832 will fix that problem. I am closing this ticket.